src/config/index.ts

```diff
@@ -103,12 +103,19 @@ export class Config {
   }
 
   public static getUserRoleConfig(contactType: ContactType): ContactProperty {
+    const parameter = contactType.user_role.reduce(
+      (acc: { [key: string]: string }, curr: string) => {
+        acc[curr] = curr;
+        return acc;
+      }, {}
+    );
+
     return {
       friendly_name: 'Role(s)',
```

Comment:
i think we only use this when there are multiple roles, so can we just use `Roles`?